src/data/store/index.js

```diff
@@ -15,14 +15,16 @@ const namespaceLabel = 'Backendless.Data.of(<ClassName>)'
 
 class DataStore {
 
-  constructor(model) {
+  constructor(model, classToTableMap) {
```

Comment:
what do you thing about renaming `classToTableMap` -> `classesMapping` ?

Reply:
Why do you think that would be better? IMO with the current version of the name the relativity of relations becomes clear.

Reply:
ok, leave this name, looks like you are right